### ISSUE TITLE
Repeat target specific information in "target details".

### ISSUE THREAD

**Issue by indolering** [HaxeFoundation/HaxeManual#460]
Users don't [read entire manuals](https://en.wikipedia.org/wiki/Minimalism_(technical_communication)) and target specific details embedded elsewhere should at least be cross-referenced in each target's subsection in chapter 12.  This is a good place to elaborate on differences in compilation switches, for example.

**Comment by Aurel300** [comment]
I agree with this, I did find the target-specific sections a bit too thin. Do you have any specific suggestions apart from the compiler defines?

**Comment by indolering** [comment]
After looking a the other targets, it appears that Java is the only one that doesn't include information about compiler switches.  You could also include a link to the language's API section.

The compiler table currently doesn't link to the language section.

Everything else I would say about targets I know about belong in the comments section.
